### ISSUE TITLE
local bind server

### ISSUE THREAD

**Issue by Caaruzo** [cbirkenbeul/docker-homelab#155]
can you please do a docker compose with configuring a bind-dns server?
Should be useable for local (internal) domains please.

thx very much

**Comment by cbirkenbeul** [comment]
So for the hole local network not just for docker?

**Comment by Caaruzo** [comment]
yes, one that could be used for the whole local network with an own internal domain like "mydomain.intern".
I already build one currently but it doesnt seems to work. Cant reach the name server. Same bind config in a virtual machine without docker works.

`version: '3.7' 

########################### SERVICES
services:
  bind:
    container_name: bind
    user: 105:106
    image: internetsystemsconsortium/bind9:9.16
    restart: unless-stopped
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /etc/timezone:/etc/timezone:ro
      - ./volume/config:/etc/bind
      - ./volume/records:/var/lib/bind
      - ./volume/cache:/var/cache/bind 
    networks:
      - default
    ports:
      - "53:53/udp" 
      - "53:53/tcp" 

########################### NETWORKS 
networks:
  default:
    driver: bridge
`

**Comment by cbirkenbeul** [comment]
I guess you have to use host hostwork. Its for other services like avahi the same.

**Comment by Caaruzo** [comment]
Thanks much, but this was not needed. it works like i did. No idea why it didnt worked before. Maybe it was just a resolving problem on windows side, ignored by ipconfig /flushdns xD.
